examples/glusterfs/README.md

```diff
@@ -75,6 +75,18 @@ NAME                ENDPOINTS
 glusterfs-cluster   10.240.106.152:1,10.240.79.157:1
 ```
 
+We should also create a service for this endpoints, so that it will be persistented. You can see [glusterfs-service.json](glusterfs-service.json) for details. The reasons we need to add the endpoints manually with the service are:
+
+* The service do not have a selector.
```

Comment:
Also this isn't the reason *why* you add the endpoints manually, this is the mechanism by which you tell the system that you *want* to add them manually.